ygen/protogen.go

```diff
@@ -437,6 +518,17 @@ func safeProtoFieldName(name string) string {
 	return replacer.Replace(name)
 }
 
+// safeProtoEnumName takes na input string which represents the name of a YANG enumeration
+// value, or identity name, and sanitises it for use in a protobuf schema.
+func safeProtoEnumName(name string) string {
```

Comment:
s/na/an
I'm getting a bit lost in the stacked up reviews but I thought I had reviewed this before and you'd added some comments about what chars are safe. Would suggest at least putting an example like My..Special/Name* -> My__Special_Name_

Reply:
I did in `safeProtoFieldName` -- looking at the grammar in 6020, these actually use the same spec, so I collapsed this function together with that one to make `safeProtoIdentifierName`.